### PR TITLE
doc(Request): Edit access_route, remote_addr docstrings

### PR DESCRIPTION
**Pull request by kgriffs**
Move the docstrings for access_route and remote_addr to the class
docstring. Edit the text for clarity.

### CODE REVIEW

File: falcon/request.py [hunk 1]
```diff
@@ -78,12 +78,49 @@ class Request(object):
...
 
-        user_agent (str): Value of the User-Agent header, or ``None`` if the
-            header is missing.
-        app (str): Name of the WSGI app (if using WSGI's notion of virtual
-            hosting).
         env (dict): Reference to the WSGI environ ``dict`` passed in from the
             server. See also PEP-3333.
+        app (str): Name of the WSGI app (if using WSGI's notion of virtual
+            hosting).
+        access_route(list): IP address of the original client, as well
```
Comment (fxfitz) [inline falcon/request.py]: Woot! I feel like I so requested this. ;-)

File: falcon/request.py [hunk 2]
```diff
@@ -78,12 +78,49 @@ class Request(object):
...
+
+                .. _RFC 7239: https://tools.ietf.org/html/rfc7239
+
+            Warning:
+                Headers can be forged by any client or proxy. Use this
+                property with caution and validate all values before
+                using them. Do not rely on the access route to authorize
+                requests.
+
+        remote_addr(str): IP address of the closest client or proxy to
```
Comment (fxfitz) [inline falcon/request.py]: I lied. Maybe it was this one. ;-)

**Comment by fxfitz** [comment]
LGTM

**Comment by philiptzou** [comment]
:+1:

**Pull request merged**